packages/load-dist/README.md

```diff
@@ -0,0 +1,43 @@
+# @loopback/load-dist
+
+Load the correct `dist` folder depending on the Node.js version we are running:
+
+version|directory
+-|-
+8.x LTS|`dist8`
+9.x|`dist8`
+10.x|`dist10`
+newer|`dist10`
+
+## Installation
+
+Run the following command to install the CLI.
+
+`npm install -g @loopback/load-dist`
+
+## Basic Use
+
+Configure your TypeScript build to produce two distribution versions:
+
+ - `dist8` compiled for `es2017` target
+ - `dist10` compiled for `es2018` target
```

Comment:
How are we going to deal with different versions of `@types/node`?

- `@types/node@8.x`
- `@types/node@10.x`

Reply:
This is a good question! IMO, we should be using `@types/node@10.x` and pay attention to any places that are different in Node.js 8.x.

Either way we choose, I feel this decision is orthogonal to this pull request. Even if we transpile everything to ES2017 target (as we are doing now, without my proposed changes), we still need to take into account API differences between different Node.js versions.